packages/compass-sidebar/src/components/csfle-connection-modal/csfle-connection-modal.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  css, spacing, Banner, Description, Label, Link, Toggle, Modal
+} from '@mongodb-js/compass-components';
+
+const toggleStyles = css({
+  marginBottom: spacing[3],
+  marginRight: spacing[3]
+});
+
+function CSFLEConnectionModal({ csfleMode, open, setOpen, setConnectionIsCSFLEEnabled }) {
+  return (<Modal
+    open={open}
+    title="Client-Side Field-Level Encryption"
+    trackingId="csfle_connection_modal"
+    setOpen={setOpen}
+  >
+    <div>
+      <p>
```

Comment:
Would the `Body` or `Subtitle` from leafygreen or one of the other `Typography` elements be better for this?
https://www.mongodb.design/component/typography/example/
```suggestion
      <Body>
```

Reply:
Sure – `Body` seems good here!